### PR TITLE
chore: HomeStack skeleton, TabNavigator

### PR DESCRIPTION
**Pull request by markdlv**
## Changes
- Skeleton of HomeScreen and HomeStack
- Skeleton of TabNavigator

I started with a TabNavigator that looked more like what we have currently, with `TabBar` and `TabBarButton` components etc.  I got a warning where react detected that we were creating components within a render.  I'm not sure/convinced that's actually true, but I went ahead and simplified to what I have in this PR.  Seeing it in its simplest form like this, I'm excited to see how far we can take it using the `bottom-tabs` lib and NOT write those `TabBar` & `TabBarButton` components - we'll see.

## Screenshots
https://user-images.githubusercontent.com/1445395/219382293-d34a8d90-3a6c-4759-a05b-1544e36e4f53.mp4

https://user-images.githubusercontent.com/1445395/219382308-34da1978-3985-4db8-9558-bb80828a24d2.mov

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,11 +37,19 @@ those versions. On the other hand, this can become cumbersome if we get into the
...
 
 If you are a LifeOmic customer, building a mobile app, and see a dependency that
 you feel should be a peer dependency, reach out to us at
 development@lifeomic.com.
 
+### Export everything, except default
+
+Try not to use `export default`. If we use named exports everywhere, we can
+`export * from ...` in index.ts files that facilitate what is available in the
+npm lib. Try to export any/everything that might be buseful for reuse, and try
```
Comment (sternetj) [inline README.md:50]:
```suggestion
npm lib. Try to export any/everything that might be useful for reuse, and try
```

File: src/screens/HomeScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        message={t('home-screen-loading', 'Loading account information')}
+      />
+    );
+  }
+
+  return (
+    <View style={styles.container} testID="home-screen">
+      <SafeAreaView>
+        <ScrollView
+          overScrollMode="always"
```
Comment (sternetj) [inline src/screens/HomeScreen.tsx:28]: Is this what we want for Android? Personally I like the overscroll behavior but I am an iOS user so I might be biased.

**Review by sternetj — APPROVED** [review]

**Review by bruce-glazier — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (markdlv) [reply README.md:50]: 👍 I'll fix this in an upcoming "cleanup" PR - trying to get these merged to stop the conflict-fest for Bruce's graphql PR.

File: src/screens/HomeScreen.tsx (hunk 2 above)
Comment (markdlv) [reply src/screens/HomeScreen.tsx:28]: Totally fair question- let's worry about this later.

**Pull request merged**